OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -360,7 +363,10 @@ public override IEnumerable<Target> GetTargets(Actor self)
 			}
 		}
 
-		class MoveFirstHalf : MovePart
+		/// <summary>
+		/// Performs the visual movement from a world position of a cell to another
```

Comment:
Move*Half updates the actor CenterPosition, which is the primary actor position in the engine.  That is far more than just visual movement.